### ISSUE TITLE
wrong remove code

### ISSUE THREAD

**Issue by Ogimle** [penumbragames/blockwars#1]
with this commit
https://github.com/penumbragames/blockwars/commit/65a47e8ad2cdb523e5c269eede4061165480e7d8
Util.almostEqual Util.linearScale methods is removed
but this methods is used in server/player.js and static/js/Player.js

**Comment by omgimanerd** [comment]
Good catch, I have no idea why I removed this. Because this project is no longer maintained and almost all the code in here has been rewritten in other forms, I will probably not be updating or fixing this. I will leave this issue here.
If you are planning on building off this project, please let me know if you have questions.
